modules/gapi/test/s11n/gapi_s11n_tests.cpp

```diff
@@ -45,6 +46,49 @@ template<> struct CompileArgTag<MyCustomType> {
 } // namespace detail
 } // namespace cv
 
+namespace {
+    class MyRMatAdapter : public cv::RMat::Adapter {
```

Comment:
No alignment between `class` (in OpenCV, namespaces don't increase the indent level)

Reply:
Fixed